Python library fails to compile on OSX 10.12.5, boost-python 1.64.0

When attempting to compile the Python library using the command

     pip install --no-cache-dir -e .
	 
from the `python` subdirectory, the following error message is received

```
    Using Python 2.7
    /usr/bin/g++ -std=c++0x -I/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/include/python2.7 -I/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/include/python2.7 -I /usr/local/include -I ../rapidjson/include -fPIC -c pylibvw.cc -o pylibvw.o
    pylibvw.cc:463:58: error: reference to non-static member function must be called; did you mean to call it with no arguments?
    double get_weighted_examples(vw_ptr vw) { return vw->sd->weighted_examples; }
                                                     ~~~~~~~~^~~~~~~~~~~~~~~~~
                                                                              ()
```

By modifying Line 463 of pylibvw.cc,

	
	double get_weighted_examples(vw_ptr vw) { return vw->sd->weighted_examples();
	
The library compiles and the python tests 

       python setup.py test

pass.

```
_________________________________________________________________________________________________ summary _________________________________________________________________________________________________
  py27: commands succeeded
  py35: commands succeeded
  congratulations :)
```

I am submitting a PR for this change.

--- a/python/pylibvw.cc
+++ b/python/pylibvw.cc
@@ -460,7 +460,7 @@ float    get_loss(example_ptr ec) { return ec->loss; }
 float    get_total_sum_feat_sq(example_ptr ec) { return ec->total_sum_feat_sq; }
 
 double get_sum_loss(vw_ptr vw) { return vw->sd->sum_loss; }
-double get_weighted_examples(vw_ptr vw) { return vw->sd->weighted_examples; }
+double get_weighted_examples(vw_ptr vw) { return vw->sd->weighted_examples(); }
 
 bool search_should_output(search_ptr sch) { return sch->output().good(); }
 void search_output(search_ptr sch, string s) { sch->output() << s; }

